src/Autosuggest.js

```diff
@@ -126,6 +130,7 @@ export default class Autosuggest extends Component {
 
   focusOnSuggestion(suggestionPosition) {
     let [sectionIndex, suggestionIndex] = suggestionPosition;
+    let suggestion = this.getSuggestion(sectionIndex, suggestionIndex);
```

Comment:
I wonder what will happen if `suggestionIndex === null`. This will happen when first suggestion is focused and Up is clicked, or the last suggestion is focused and Down is clicked. In this case we shouldn't be calling `onSuggestionFocused`, I think.